plugins/clipboard/plugin.js

```diff
@@ -363,6 +363,25 @@
 					}, 0 );
 				}
 			}, null, null, 1000 );
+
+			// Instert fake element from external data transfer (#638).
+			editor.on( 'paste', function( evt ) {
```

Comment:
I'm wondering if it shouldn't be moved into `fakeobjects` plugin as without it there is no way to insert such element into the editor. In that case tests also could be moved.